keras_cv/metrics/coco/pycoco_wrapper.py

```diff
@@ -152,7 +152,7 @@ def _convert_groundtruths_to_coco_dataset(groundtruths, label_map=None):
     gt_annotations = []
     num_batches = len(groundtruths["source_id"])
     for i in range(num_batches):
-        max_num_instances = groundtruths["classes"][i].shape[1]
+        max_num_instances = max(x.shape[0] for x in groundtruths["classes"][i])
```

Comment:
How about we check if this is ragged, and if it is we use `groundtruths["classes"][i].row_lengths(axis=1)`

Reply:
@ianstenbit 
The computation of `_convert_groundtruths_to_coco_dataset` is in numpy. I have updated the ragged/dense status check by utilizing the shape of the numpy array.

Reply:
Ah yes sorry I forgot this -- I think how you had it before is better because this new version should behave the same way but is a bit more complex.

Reply:
I have reverted.